README.md

```diff
@@ -69,7 +69,7 @@ By default you create a [bare-workflow React](https://docs.expo.io/bare/explorin
 
 ## Usage with Expo Client App
 
-Expo Client enables you to work with all of the [Components and APIs](https://facebook.github.io/react-native/docs/getting-started.html) in `react-native`, as well as the [JavaScript APIs](https://docs.expo.io/versions/latest/sdk/index.html) that the are bundled with the Expo App.
+Expo Client enables you to work with all of the [Components and APIs](https://facebook.github.io/react-native/docs/getting-started) in `react-native`, as well as the [JavaScript APIs](https://docs.expo.io/versions/latest/sdk/index) that the are bundled with the Expo App.
```

Comment:
The correct link to latest Expo docs is https://docs.expo.io/versions/latest/.

Before the change (with `.html) link cause a redirect to the correct page, now it leads to an error page:
<img width="437" alt="Screenshot 2020-11-02 163900" src="https://user-images.githubusercontent.com/719641/97887351-126e9100-1d2a-11eb-9ff0-85e6cd3484b5.png">

Reply:
Ah, thanks! Should I revert it or update it to the newer link?

Reply:
I'm not the CRNA maintainer but I would recommend updating the Expo link to the new format. 🙂